linux_proc_extras/datadog_checks/linux_proc_extras/linux_proc_extras.py

```diff
@@ -24,10 +24,12 @@
 
 
 class MoreUnixCheck(AgentCheck):
-    def check(self, instance):
-        self.tags = instance.get('tags', [])
+    def __init__(self, *args, **kwargs):
+        super(MoreUnixCheck, self).__init__(*args, **kwargs)
+        self.tags = self.instances[0].get('tags', [])
         self.set_paths()
```

Comment:
`set_paths` uses `self.agentConfig`. Isn't that deprecated?

Reply:
Yes I think it is, we should use `datadog_agent.get_config` instead. It doesn't even look like `self.agentConfig` contains anything anymore 🤔 

https://github.com/DataDog/datadog-agent/blob/master/pkg/collector/python/check.go#L199-L218